backend/src/database/migrations/V1719390926__orgAvgContributorEngagement.sql

```diff
@@ -0,0 +1,16 @@
+ALTER TABLE "organizationSegmentsAgg" ADD COLUMN "avgContributorEngagement" INTEGER DEFAULT '-1'::INTEGER;
```

Comment:
Let's do it this way:
1. Have the field non-nullable like every other aggregated field in this table
2. So, no arbitrary default values, which if they come up at some point in the UI, would look confusing
3. So, drop the default value here
4. In the end of the migration, after populating values, make the field non-nullable

Reply:
Quick question: If we make it non-nullable, shouldn’t we handle it in the code if the aggregation is null? 

For example, if the `avgContributorEngagement` is null for an organization and segment, can we use COALESCE and fallback to 0?

Reply:
If there are activities, aggregation can’t be null. If there are no activities, the org/segment will be excluded from the query.